Website/Website/auth.js

```diff
@@ -89,6 +89,10 @@ function initialize(host) {
     const authContext = new AuthenticationContext(`https://login.microsoftonline.com/${tenantName}`);
     const serviceClusterUrl = env.serviceClusterUrl;
 
+    // Updates will be made once the ARM support is added.
+    // This expects a secret with a value that is a json object comprising of all the backend service names and the external IP addresses as exposed by AKS where each of them can be listened to.
+    const kubernetesServices = env.kubernetesServices ? env.kubernetesServices : null;
```

Comment:
can this be just like this?
const kubernetesServices = env.kubernetesServices

Reply:
why do you even need to define a separate constant variable kubernetesServices? You know you can access env.kubernetesServices later in the code, right? I'd suggest removing this unnecessary variable.

Reply:
Removed.